example/src/services/HearthstoneService.js

```diff
@@ -0,0 +1,19 @@
+import { wrapService, withPostSuccess } from 'redux-recompose';
+
+import responseBody from './response';
+
+// Declare your api calls
+const getCards = async () => new Promise(resolve => setTimeout(() => resolve(responseBody), 1000));
+
+// Declare your customizations, used by fetchMiddleware
+getCards.successSelector = response => response.cards;
+getCards.injections = [
+  withPostSuccess((dispatch, response, state) => alert(`Fetched at: ${state.hearthstone.count}`))
```

Comment:
🤔 It's a good place to handle the extra or custom behavior of a service? Wouldn't confuse the dev a bit with the flow of the action in this place?

Reply:
I'm not sure the service layer is where we should be doing pre/post fetch actions, or anything related to the store actually

Reply:
>  It's a good place to handle the extra or custom behavior of a service? Wouldn't confuse the dev a bit with the flow of the action in this place?

IMHO yes, because we are declaring how these usually behaves after we do an API request. This kind of expressions let the dev to express the behavior of these in a single file.
If we need to distinguish them of traditional reducers, I'd give this another name, like `Dispatcheable Service`.